packages/recommend-react/src/RelatedProducts.tsx

```diff
@@ -1,6 +1,7 @@
+import { GetRelatedProductsProps } from '@algolia/recommend-core';
 import {
   createRelatedProductsComponent,
-  RelatedProductsProps,
+  RelatedProductsProps as RelatedProductsVDomProps,
```

Comment:
```suggestion
  RelatedProductsProps as RelatedProductsVDOMProps,
```

Since it is the official casing and we capitalize it in the repo, I think it make sense to also do it for the types

Reply:
See e95320836fcda0206607e213f1a85a1685e684f9.